### ISSUE TITLE
Team based learning paths

### ISSUE THREAD

**Issue by open-sudo** [redhat-developer/rhdh#888]
### What do you want to improve?
Currently, learning paths in DH are static content populated from a json file. It would be great to have flexibility such that teams or team members could have learning paths specific to them. The frontend team could have reactjs in their learning paths, the platform team could have CircleCI in their learning paths while the Java may have as objectif to learn Quarkus.

### What is the current behavior?
Currently, the learning path is pulled from a single static file.

### What is the new behavior?
The new behavior should give teams the possibility to define learning paths for their specific group. The PM could also be responsible for creating learning paths for the team. The key here is the capability to have different learning paths active at the same time. A user could have many different learning paths depending on the teams that they are in. I do not know if we want to make this so granular that each individual user has their own learning path. But in my mind team-based would be great.

**Comment by rafaeltuelho** [comment]
Hi team!
Any update on the roadmap for this capability?

**Comment by invincibleJai** [comment]
@rafaeltuelho / @open-sudo  there is a dedicated feature for this being tracked in Jira https://issues.redhat.com/browse/RHIDP-476, but this is yet to be panned. 

The above issue talks about two things

1. Making the Learning path dynamic i.e configurable over static one (as we have for home/tech-radar)
2. Tailored Learning path for each team/org

point 2 can be handled with the feature linked above. However, point 1 we can handle as a separate item if it helps. Let us know your thoughts 


cc @christophe-f @kadel @rtaniwa

**Comment by rafaeltuelho** [comment]
Yeah, I believe the 1st step would be just adding the ability to declaratively create a list of learning paths using some `data.json` mechanism used by the home and tech-radar.

**Comment by invincibleJai** [comment]
> Yeah, I believe the 1st step would be just adding the ability to declaratively create a list of learning paths using the same `data.json` mechanism used by the home and tech-radar.

This is tracked over here  https://issues.redhat.com/browse/RHIDP-1638
